hadoop-connector-examples/README.md

```diff
@@ -101,3 +101,45 @@ You can also use hadoop-connectors on Spark
 Spark (verified with Spark 2.2.1 on Ubuntu 16.04)
    spark-submit --class io.pravega.examples.spark.WordCount ${HADOOP_EXAMPLES_JAR} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
 ```
+
+## Terasort
+
+Hadoop (verified with Hadoop 2.8.1 and 3.1.1 on Ubuntu 16.04)
+
+### Execution
+
+```
+1. setup and start hdfs as well as pravega
+
+2. set env variables
+   export HDFS=hdfs://<hdfs_ip_and_port> # e.g. hdfs://192.168.0.188:9000
+   export HADOOP_EXAMPLES_JAR=<pravega-hadoop-examples-0.4.0-SNAPSHOT-all.jar location> # e.g. ./build/libs/pravega-hadoop-examples-0.4.0-SNAPSHOT-all.jar
```

Comment:
Here is still `pravega-hadoop-examples-0.3.0-SNAPSHOT-all.jar` instead of `pravega-hadoop-examples-x.y.z.jar`.